samcli/local/docker/container.py

```diff
@@ -71,6 +72,7 @@ def __init__(
         :param docker_client: Optional, a docker client to replace the default one loaded from env
         :param container_opts: Optional, a dictionary containing the container options
         :param additional_volumes: Optional list of additional volumes
+        :param string container_host: Optional. Host of locally emulated Lambda container
```

Comment:
this cannot be optional anymore, because if customers do not specify a value, click will use the default value `localhost`.

Reply:
Same reason as my another comment below.